### DISCUSSION TITLE
Use Two Themes for Filament

### DISCUSSION THREAD

**Discussion by falconsmilie** [filamentphp/filament#5760] [Help] [answered]
Hi.

We are currently using Filament admin and are happy with the default theme. This is basically our CMS for our application.

We are considering using the `Forms` builder on our user facing public frontend (because we love the ease of use) and are wondering if it's possible to keep the default styling for the admin, and have another style for the frontend `Forms` builder. 

Same for `Notifications`.

**Comment by danharrin** [comment, marked as answer]
Yes, you can use CSS to customise the style of our components. We have many `filament-*` classes you can hook into
